mldsa_parse.py

```diff
@@ -1,11 +1,3 @@
-#!/usr/bin/env python
```

Comment:
It's in the wrong place and it's an older version -- the copyright header was added later from memory.

Reply:
Should be reverted now.